datafusion/physical-plan/src/joins/sort_merge_join.rs

```diff
@@ -2910,6 +2984,310 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn join_right_anti_one() -> Result<()> {
```

Comment:
what one means here?

Reply:
The meaning is "a column." This naming has already been updated.